vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/BiblioBlock.java

```diff
@@ -0,0 +1,83 @@
+package edu.asu.diging.vspace.core.model.impl;
+
+import javax.persistence.Entity;
+import javax.persistence.Transient;
+
+import org.commonmark.node.Node;
+import org.commonmark.parser.Parser;
+import org.commonmark.renderer.html.HtmlRenderer;
+
+import edu.asu.diging.vspace.core.model.IBiblioBlock;
+
+@Entity
+public class BiblioBlock extends ContentBlock implements IBiblioBlock {
+
+    private String title;
+    
+    private String author;
+    
+    private Integer year;
+    
+    private String journal;
```

Comment:
for now, let's add at least the following:
- url 
- volume
- issue
- pages
- editors
- type
- note

we'll need to come up with a more flexible schema at some point, but let's leave it with this for now